media-plugins/gst-plugins-qt5/gst-plugins-qt5-1.18.4.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
empty line after EAPI please.

Reply:
Added. Format was copied from another GStreamer ebuild.

Reply:
Absolutely all have no newline, so neither should it have it here

Reply:
Another style difference between gnome/gstreamer and everyone else.